File: KingdomCodeTool/wwwroot/Download/202501201119/Angular/shared/Sys_Computers.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sys_Computers } from './Sys_Computers.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sys_ComputersService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Computer_Id', 'ComputerName', 'IPAddress', 'LastLoginTime', 'IsLogin', 'LoginTime', 'UserName', 'UserDomainName'];

    List: Sys_Computers[] | undefined;
    ListFilter: Sys_Computers[] | undefined;
    FormData!: Sys_Computers;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sys_Computers";
    }
}

File: KingdomCodeTool/wwwroot/Download/202501201119/Service/Implement/Sys_ComputersService.cs
namespace Service_eHospital_DongNai_A_System.Implement
{
    public class Sys_ComputersService : BaseService<Sys_Computers, ISys_ComputersRepository>
    , ISys_ComputersService
    {
    private readonly ISys_ComputersRepository _Sys_ComputersRepository;
    public Sys_ComputersService(ISys_ComputersRepository Sys_ComputersRepository) : base(Sys_ComputersRepository)
    {
    _Sys_ComputersRepository = Sys_ComputersRepository;
    }
    }
    }

